specification/sections/01-flag-evaluation.md

```diff
@@ -104,25 +107,38 @@ See [hooks](./04-hooks.md) for details.
 OpenFeature.getProviderMetadata();
 ```
 
-See [provider](./02-providers.md) for details.
+It must be possible to access provider metadata using a `domain`.
+If a provider has not be registered under the requested domain, the default provider metadata is returned.
+
+```typescript
+// example provider accessor
+OpenFeature.getProviderMetadata("domain-1");
+```
+
+See [provider](./02-providers.md), [domain](../glossary.md#domain) for details.
 
 ### Creating clients
 
 #### Requirement 1.1.6
 
 > The `API` **MUST** provide a function for creating a `client` which accepts the following options:
 >
-> - name (optional): A logical string identifier for the client.
+> - domain (optional): A logical string identifier for binding clients to provider.
 
 ```java
 // example client creation and retrieval
-OpenFeature.getClient("my-named-client");
+OpenFeature.getClient();
 ```
 
-The name is a logical identifier for the client which may be associated with a particular provider by the application integrator.
-If a client name is not bound to a particular provider, the client is associated with the default provider.
+It must be possible to create a client that is associated with a `domain`.
```

Comment:
```suggestion
It's possible to create a client that is associated with a `domain`.
```

```suggestion
It's possible to create a client that is associated with a `domain`.
```

I try to avoid must/should/etc (RFC2119 language) in non-normative sections. I recommend speaking as if things are already implemented in these sections (like the sentence below: "the client will..."